src/sensors/distance/infrared/analog/sharp/GP2Y0A21.cpp

```diff
@@ -17,8 +17,6 @@ unsigned int GP2Y0A21::getDistance()
 {
     auto analogReading = mRuntime.getAnalogPinState(kPin);
     // Formula source: Jeroen Doggen http://plat.is/v3x25
-    // It's OK to surpress this clang-tidy warning since this is part of a magic (!) formula
-    // NOLINTNEXTLINE(readability-magic-numbers)
     auto result = static_cast<unsigned int>(1 / (0.0002391473 * analogReading - 0.0100251467));
```

Comment:
:warning: **readability-magic-numbers** :warning:
0.0002391473 is a magic number; consider replacing it with a named constant